app/buyers/views/buyers.py

```diff
@@ -246,3 +249,54 @@ def delete_a_brief(framework_slug, lot_slug, brief_id):
             url_for('.view_brief_summary', framework_slug=framework_slug, lot_slug=lot_slug,
                     brief_id=brief_id, delete_requested=True)
         )
+
+
+@buyers.route("/buyers/frameworks/<framework_slug>/requirements/<lot_slug>/<brief_id>/answer-question",
+              methods=["GET", "POST"])
+def add_clarification_question(framework_slug, lot_slug, brief_id):
+
+    framework, lot = get_framework_and_lot(framework_slug, lot_slug, data_api_client,
+                                           status="live", must_allow_brief=True)
+
+    brief = data_api_client.get_brief(brief_id)["briefs"]
+    if not is_brief_associated_with_user(brief, current_user.id):
+        abort(404)
+
+    if brief["status"] != "live":
+        abort(404)
+
+    if not clarification_questions_open(brief):
+        abort(404)
+
+    content = content_loader.get_manifest(framework_slug, "clarification_question")
+    section = content.get_section(content.get_next_editable_section_id())
+
+    errors = {}
+    status_code = 200
+
+    if request.method == "POST":
+        try:
+            data_api_client.add_brief_clarification_question(brief_id,
+                                                             request.form.get("question", ""),
```

Comment:
Although I suppose it is following the same pattern as elsewhere to let the API do all validation, so perhaps this is right.